packages/block-library/src/list-item/edit.js

```diff
@@ -122,17 +122,18 @@ function useOutdentListItem( clientId ) {
 		[ clientId ]
 	);
 	const { replaceBlocks, selectionChange } = useDispatch( blockEditorStore );
+	const {
+		getBlockRootClientId,
+		getBlockAttributes,
+		getBlock,
+		getBlockIndex,
+		getSelectionStart,
+		getSelectionEnd,
```

Comment:
I think these now have to be added to the dependency array of the `useCallback` call below? Or are the references guaranteed to be stable similar to the `setXXX` callbacks returned by `useState`?

Reply:
Correct. References a guaranteed to be stable, similar to core hooks.